app/src/main/java/com/sdp13epfl2021/projmag/activities/listenerClass/RecyclerItemClickListenr.kt

```diff
@@ -0,0 +1,48 @@
+package com.sdp13epfl2021.projmag.activities.listenerClass
+
+import android.content.Context
+import android.view.GestureDetector
+import android.view.MotionEvent
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+
+class RecyclerItemClickListenr(context: Context, recyclerView: RecyclerView, private val mListener: OnItemClickListener?) : RecyclerView.OnItemTouchListener {
```

Comment:
typo RecyclerItemClickListenr -> RecyclerItemClickListen**e**r

Reply:
Not a typo, I wanted to be cool :)

Reply:
I see, you are probably a geoguessr gamer XD